NobankCore/noBank/src/main/java/com/hunk/nobank/activity/dashboard/DashboardActivity.java

```diff
@@ -0,0 +1,61 @@
+package com.hunk.nobank.activity.dashboard;
+
+import android.os.Bundle;
+import android.support.annotation.VisibleForTesting;
+import android.widget.TextView;
+
+import com.hunk.nobank.R;
+import com.hunk.nobank.activity.BaseActivity;
+import com.hunk.nobank.contract.Money;
+
+public class DashboardActivity extends BaseActivity implements DashboardView {
+    private DashboardPresenter mPresenter;
+    @VisibleForTesting
+    private TextView mBalance;
+
+    @Override
+    protected boolean isRequiredLoginedUserSession() {
+        return true;
+    }
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_dashboard_page, Base.NORMAL);
+        // set Title
+        getTitleBarPoxy().getTitle().setText(R.string.dashboard);
+        mBalance = (TextView) findViewById(R.id.txt_balance);
+        //
+        mPresenter = new DashboardPresenter(this);
+    }
+
+    @Override
+    protected void onResume() {
+        super.onResume();
+        mPresenter.onResume();
+    }
+
+    @Override
+    protected void onDestroy() {
+        mPresenter.onDestroy();
```

Comment:
Create a baseMVPActivity and basePresenter class to do a common presenter.onDestroy().